src/hooks/useStatusV2.ts

```diff
@@ -0,0 +1,15 @@
+import { useQuery, UseQueryResult } from '@tanstack/react-query'
```

Comment:
Foreløpig er planen å fetche dataen på server side så vi trenger ikke en client side fetch for nå

Reply:
Kan godt det, var mest for å teste at dataene kommer frem. Kan godt slette